src/live.rs

```diff
@@ -82,7 +85,13 @@ pub fn iso_ignition_embed(config: &IsoIgnitionEmbedConfig) -> Result<()> {
     // write new config
     embed.seek_to_start()?;
     embed.write(&cpio)?;
-    holder.complete();
+    if config.output.as_deref() == Some("-") {
+        stdout().write_all(&cpio).chain_err(|| "writing output")?;
+        stdout().flush().chain_err(|| "flushing output")?;
+        drop(holder)
```

Comment:
This doesn't save any disk I/O, though.  We're still copying the image to a temporary file (using reflinks to avoid I/O, if reflinks are available) and then deleting it afterward.

Reply:
There are now 2 commits on this PR. The first uses the easiest method (creating the temporary file and deleting afterward), the second skips the holder.

Reply:
Looking into eliminating the need for the temp file I got the feeling I would need to, at a minimum, add new methods to EmbedArea. Do you have any thoughts/guidance on pursuing this route?